TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -1582,6 +1594,14 @@ and a throwing method can't satisfy a protocol requirement for a rethrowing meth
 That said, a rethrowing method can override a throwing method,
 and a rethrowing method can satisfy a protocol requirement for a throwing method.
 
+<!-- XXX comparison between rethrows and generic typed throws 
```

Comment:
Part of me wants to suggest that this whole section be revised to replace `rethrows` entirely with the use of generic typed throws, because that's how I expect the language to go over time. Then, `rethrows` would become more of a historical anecdote. However, that is a lot of work, and perhaps we'll find that `rethrows` will remain for longer than I expect. If so, I think it's fine to keep this comparison short: take the `someFunction` example from the beginning of the section and make it generic over the thrown error type, and note that it's providing a more precise guarantee about what errors it rethrows than `rethrows` does (because `rethrows` always produces an `any Error`).